### PR TITLE
feature: add fixed reviewer config

### PR DESCRIPTION
**Pull request by chisacam**
## Summary by CodeRabbit

- New Features
  - Support optional fixed reviewers that are always included; dynamic reviewers fill remaining slots. PR creators are excluded from reviewer pools; returns all when requested count ≥ available, and handles empty/edge cases gracefully.

- Tests
  - Test setup simplified: shared reviewer selector instance and explicit per-test mock setup to reflect new behaviors.

- Documentation
  - README and README.en examples updated to show separate optional "reviewers" and "fixedReviewers" pools.

- Chores
  - Package version bumped to 1.1.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,17 @@
...
+# 랜덤 리뷰어 풀 (옵션)
 reviewers:
   - githubName: '김개발'
     slackMention: '<@U1234567>' # Slack 사용자 ID
   - githubName: '박코딩'
     slackMention: '<@U2345678>' # Slack 사용자 ID
+
+# 필수 리뷰어 풀 (옵션)
+fixedReviewers:
+  - githubName: '유다희'
```
Comment (gnoyes-mik) [inline README.md:36]: @chisacam 
you-die, 유다희  ㅋㅋㅋ 너무 재밌습니다

**Review by gnoyes-mik — APPROVED** [review]
👍
